lib/client-assets.php

```diff
@@ -376,10 +383,24 @@ function gutenberg_register_scripts_and_styles() {
 	wp_register_script(
 		'wp-nux',
 		gutenberg_url( 'build/nux/index.js' ),
-		array( 'wp-element', 'wp-components', 'wp-compose', 'wp-data', 'wp-i18n', 'lodash' ),
+		array(
+			'lodash',
+			'wp-components',
+			'wp-compose',
+			'wp-data',
+			'wp-data-plugin-persistence',
+			'wp-element',
+			'wp-i18n',
+		),
 		filemtime( gutenberg_dir_path() . 'build/nux/index.js' ),
 		true
 	);
+	wp_add_inline_script(
+		'wp-nux',
+		'wp.data.use( wp.dataPluginPersistence.createPersistencePlugin( "WP_EDIT_POST_DATA_" + window.userSettings.uid ) );',
```

Comment:
Should we do the same in the `editor` module? or should we leave this for `edit-post` only?